templates/plugin/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+    "extends": "fastify-tsconfig",
+    "compilerOptions": {
+      "outDir": "dist"
```

Comment:
We don't need this outDir since this is a template for JS plugin that has also TS typings.

Reply:
We can still remove this outDir and add noEmit since this is needed only for standard